src/Pages/SubsitesVirtualPage.php

```diff
@@ -21,139 +19,68 @@
 
 class SubsitesVirtualPage extends VirtualPage
 {
-
     private static $table_name = 'SubsitesVirtualPage';
 
     private static $description = 'Displays the content of a page on another subsite';
 
-    private static $db = [
-        'CustomMetaTitle' => 'Varchar(255)',
-        'CustomMetaKeywords' => 'Varchar(255)',
-        'CustomMetaDescription' => 'Text',
-        'CustomExtraMeta' => 'HTMLText'
-    ];
-
     private static $non_virtual_fields = [
         'SubsiteID'
     ];
 
     public function getCMSFields()
     {
-        $fields = parent::getCMSFields();
-
-        $subsites = DataObject::get(Subsite::class);
-        if (!$subsites) {
-            $subsites = new ArrayList();
-        } else {
-            $subsites = ArrayList::create($subsites->toArray());
-        }
-
-        $subsites->push(new ArrayData(['Title' => 'Main site', 'ID' => 0]));
-
-        $fields->addFieldToTab(
-            'Root.Main',
-            DropdownField::create(
-                'CopyContentFromID_SubsiteID',
-                _t(__CLASS__ . '.SubsiteField', 'Subsite'),
-                $subsites->map('ID', 'Title')
-            )->addExtraClass('subsitestreedropdownfield-chooser no-change-track'),
-            'CopyContentFromID'
-        );
-
-        // Setup the linking to the original page.
-        $pageSelectionField = SubsitesTreeDropdownField::create(
-            'CopyContentFromID',
-            _t('SilverStripe\\CMS\\Model\\VirtualPage.CHOOSE', 'Linked Page'),
-            SiteTree::class,
-            'ID',
-            'MenuTitle'
-        );
-
-        $fields->addFieldToTab(
-            'Root.Main',
-            TreeDropdownField::create('CopyContentFromID', 'Linked Page', SiteTree::class)
-        );
```

Comment:
This field, which is now auto-scaffolded, is moved to the top which is right next to the subsite selector field and matches where it sits in the normal virtual page.